tools/builders/packager/schema.json

```diff
@@ -16,18 +16,38 @@
     "styleFolders": {
       "type": "array",
       "description": "Style folders that should be part of the release",
-      "default": ["style"],
+      "default": ["components/style"],
       "items": {
         "type": "string"
       }
     },
     "assetFolders": {
       "type": "array",
       "description": "Asset folders that should be part of the release",
-      "default": ["assets"],
+      "default": ["components/assets"],
       "items": {
         "type": "string"
       }
+    },
+    "lintingRulesTsConfig": {
+      "type": "string",
+      "default": "tools/linting/tsconfig.json",
```

Comment:
I would not default this in case, in one month nobody knows that this location is mentioned here. If we want to move it once we won't look up this file.